xlml/apis/test_config.py

```diff
@@ -510,13 +541,3 @@ def from_pytorch(test_name: str, network='default', subnetwork='default'):
   @property
   def benchmark_id(self) -> str:
     return self.test_name
-
-  # HACK: setup script is used as the entrypoint in the test. Make sure it
-  # invokes the content of `test_script` at the end (e.g. "${@:0}").
-  @property
```

Comment:
Did you mean to delete these?

Reply:
Hah, I think you left that comment before.

Reply:
Do you want me to delete it?

Reply:
Missed the inheritance. Did you correct the weird hack I noted where the setup_script invokes the test script? If not, can you preserve that comment as a warning?

Reply:
I cleaned up the xxxGpuTest into `GpuGkeTest`. It looks cleaner now.